### ISSUE TITLE
Group the generated events from the OpenAPI by the owning service

### ISSUE THREAD

**Issue by hartmanjan1970** [event-catalog/eventcatalog#1065]
### Use Case

In the UI view http://localhost:3000/docs under topic **Queries ()** the number of queries is now a large list of all queries from all the OpenAPI contracts from all our services.

We have multiple services with there own OpenAPI contract.
In every service we use the same query pattern, therefor the queries have the same name. 
Problem arise when we generate with OpenAPI generator the Queries, the 2nd query will override the first one.

### Proposed Solution

Would it be possible to arrange the queries by the services the contracts belongs to?
queries\{service name a}\{the queries for this service}
queries\{service name b}\{the queries for this service}

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.
* If this issue is labeled **needs-discussion**, it means the spec has not been finalized yet. Please reach out in the [EventCatalog Discord](https://discord.com/invite/3rjaZMmrAm).

**Comment by boyney123** [comment]
Yeah good points, I wonder if we should enable a new sidebar configuration folks can opt into.

At the moment the sidebar lists things by resource type as you mentioned, but maybe we could group them by domain/service too?

Example

```
My Domain
  - Service A
      - Events
      - Queries
      - Commands
  - Service B
     - Events

My Second Domain
  - Service C
  - Service D
  
etc....
```

Would this kinda of navigation be better?

**Comment by hartmanjan1970** [comment]
I think your example would be good for us

**Comment by boyney123** [comment]
The OpenAPI now groups by service when running the plugin. The #1132 feature should help with the UI and docs when thats ready

**Comment by boyney123** [comment]
Now live https://www.eventcatalog.dev/docs/api/config#docssidebar-options

Let us know what you think @hartmanjan1970
